### ISSUE TITLE
fix `releases.yml` workflow version number

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/DDRSS#6]
### Description

Update the 2 actions to the latest version

```diff
- uses: actions/checkout@v2
+ uses: actions/checkout@v3
```

```diff
- uses: TriPSs/conventional-changelog-action@v3.7.1
+ uses: TriPSs/conventional-changelog-action@v3
```

File path - https://github.com/Pradumnasaraf/DDRSS/blob/main/.github/workflows/releases.yml

**Comment by mehrdadbn9** [comment]
please assign it to me i'm eager to start contribute at community.

**Comment by Pradumnasaraf** [comment]
Yes, go ahead @mehrdadbn9

**Comment by mehrdadbn9** [comment]
@Pradumnasaraf  as you know it was easy one if even this kind of issue 's policy is to be open let me know.

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf as you know it was easy one if even this kind of issue 's policy is to be open let me know.

Sorry didn't get you

**Comment by kumarankit999** [comment]
Should I work on this, if you assign me

**Comment by Pradumnasaraf** [comment]
> Should I work on this, if you assign me

This issue is still assigned to someone, so I will give some time to them to fix it. You can look for other issues

**Comment by kumarankit999** [comment]
ok

**Comment by Pradumnasaraf** [comment]
Are you still working on this issue @mehrdadbn9?

**Comment by helloabhii** [comment]
Hey @Pradumnasaraf is it still open ?

**Comment by Pradumnasaraf** [comment]
FIXED  in 18a9804f7b670127b7b8abb211d5c208de66b181
